src/rust/ide/view/graph-editor/src/component/node/error.rs

```diff
@@ -39,3 +47,89 @@ impl Error {
+// === Error Visualization Container ===
+// =====================================
+
+// === Constants ===
+
+const SIZE          : (f32,f32) = super::super::visualization::container::DEFAULT_SIZE;
+
+
+// === Container ===
+
+/// The container containing just the error visualization and background.
+#[derive(Clone,CloneRef,Debug)]
+pub struct Container {
+    logger         : Logger,
+    visualization  : error_visualization::Error,
+    scene          : Scene,
+    // TODO : We added a HTML background to the `View`, because "shape" background was
+    //     overlapping the DOM created by error visualization. This should be further
+    //     investigated while fixing rust visualization displaying. (#796)
+    background_dom : DomSymbol,
+    display_object : display::object::Instance,
+}
+
+impl Deref for Container {
+    type Target = error_visualization::Error;
+
+    fn deref(&self) -> &Self::Target { &self.visualization }
+}
+
+impl Container {
+    /// Constructor of error container.
+    pub fn new(scene:&Scene) -> Self {
+        let scene          = scene.clone_ref();
+        let logger         = Logger::new("error::Container");
+        let display_object = display::object::Instance::new(&logger);
+        let background_dom = Self::create_background_dom(&logger,&scene);
+        let visualization  = error_visualization::Error::new(&scene);
+
+        display_object.add_child(&background_dom);
+        display_object.add_child(&visualization);
+
+        Self {logger,scene,display_object,background_dom,visualization}
+    }
+
+    fn create_background_dom(logger:&Logger, scene:&Scene) -> DomSymbol {
+        // FIXME : StyleWatch is unsuitable here, as it was designed as an internal tool for shape
+        //     system (#795)
+        let styles   = StyleWatch::new(&scene.style_sheet);
+        let bg_color = styles.get_color(ensogl_theme::graph_editor::visualization::background);
+        let bg_red   = bg_color.red*255.0;
+        let bg_green = bg_color.green*255.0;
+        let bg_blue  = bg_color.blue*255.0;
+        let bg_hex   = format!("rgba({},{},{},{})",bg_red,bg_green,bg_blue,bg_color.alpha);
+
+        let div            = web::create_div();
+        let background_dom = DomSymbol::new(&div);
+        let (width,height) = SIZE;
+        let width          = format!("{}.px", width);
+        let height         = format!("{}.px", height);
+        background_dom.dom().set_style_or_warn("width"        ,width ,&logger);
+        background_dom.dom().set_style_or_warn("height"       ,height,&logger);
+        background_dom.dom().set_style_or_warn("z-index"      ,"1"   ,&logger);
+        background_dom.dom().set_style_or_warn("overflow-y"   ,"auto",&logger);
+        background_dom.dom().set_style_or_warn("overflow-x"   ,"auto",&logger);
+        background_dom.dom().set_style_or_warn("background"   ,bg_hex,&logger);
+        background_dom.dom().set_style_or_warn("border-radius","14px",&logger);
```

Comment:
`z-index` and `border-radius` seem like they should refer to already existing constants.